tests/nvector_cloneempty.cc

```diff
@@ -0,0 +1,14 @@
+#include "nvector_eigen.h"
+#include "eigen3/Eigen/Dense"
+#include <iostream>
+
+using Vector = Eigen::Matrix<realtype, Eigen::Dynamic, 1>;
+
+int main ()
+{
+  N_Vector v = create_eigen_nvector<Vector>(2);
+
+  N_Vector x = v->ops->nvcloneempty(v);
+  // FIXME I'm not sure how to test this
+  std::cout << "help plz";
```

Comment:
I don't know the semantics of `nvcloneempty`, but I suspect that it should be implemented in such a way that you end up with an Eigen vector wrapped into an `N_Vector`. You can then extract from `x` this wrapped vector and output it, like in the other tests.

Reply:
`nvcloneempty` should have content unallocated, so I think `x->content` should be a `nullptr`.